Sources/NostrSDK/Events/Tags/EventTag.swift

```diff
@@ -96,6 +96,16 @@ public struct EventTag: RelayProviding, RelayURLValidating, Equatable {
         return EventTagMarker(rawValue: tag.otherParameters[1])
     }
 
+    /// The pubkey of the author of the referenced event.
+    public var pubkey: String? {
```

Comment:
`pubkey` was added a few months ago to text note event tags.
https://github.com/nostr-protocol/nips/commit/d68899881c647bc031c4e62fd0501323e8bad7f8